utils/http.go

```diff
@@ -16,14 +16,15 @@ type RequestResponseStatus string
 
 // RequestResponse represents both the outgoing request and response for a particular URL/method/body
 type RequestResponse struct {
-	Method     string
-	URL        string
-	Status     RequestResponseStatus
-	StatusCode int
-	Request    string
-	Response   string
-	Body       []byte
-	Elapsed    time.Duration
+	Method        string
+	URL           string
+	Status        RequestResponseStatus
+	StatusCode    int
+	Request       string
+	Response      string
+	Body          []byte
+	ContentLength string
```

Comment:
Feels like this should be an int (perhaps with a value of -1 if unset)

Reply:
not sure how to set -1 in that case.. should I change RequestResponse{} to always be RequestResponse{ContentLength: -1}?

Reply:
Yes, and should stay -1 if there is no ContentLength header